test/system/e2e_test.py

```diff
@@ -25,10 +25,11 @@ def temp_test_dir():
         yield t
 
 
-POP1_PHENO1_FILE = os.path.abspath(os.path.join(data_dir, 'two_pop/pop1_pheno1_sumstats.txt'))
-POP2_PHENO1_FILE = os.path.abspath(os.path.join(data_dir, 'two_pop/pop2_pheno1_sumstats.txt'))
-POP1_POP2_LDSCORE_FILE = os.path.abspath(os.path.join(data_dir,
-                             'two_pop/pop1_pop2_chr1.l2.ldscore'))
+TWO_POP_DIR = os.path.abspath(os.path.join(data_dir, 'two_pop'))
+POP1_PHENO1_FILE = os.path.abspath(os.path.join(TWO_POP_DIR, 'pop1_pheno1_sumstats.txt'))
+POP2_PHENO1_FILE = os.path.abspath(os.path.join(TWO_POP_DIR, 'pop2_pheno1_sumstats.txt'))
+POP1_POP2_LDSCORE_FILE = os.path.abspath(os.path.join(TWO_POP_DIR, 'pop1_pop2_chr1.l2.ldscore'))
```

Comment:
Does this work ok without the `.gz`?

Reply:
It seems to, yes.  That's the name of the file in the test data directory.